src/openrct2/windows/map.c

```diff
@@ -1349,41 +1347,32 @@ static void window_map_place_park_entrance_tool_down(sint32 x, sint32 y)
  */
 static void window_map_set_peep_spawn_tool_down(sint32 x, sint32 y)
 {
-    rct_map_element *mapElement, *surfaceMapElement;
+    rct_map_element *mapElement;
     sint32 mapX, mapY, mapZ, direction;
 
     footpath_get_coordinates_from_pos(x, y, &mapX, &mapY, &direction, &mapElement);
-    if (mapX == 0x8000)
-        return;
-
-    surfaceMapElement = map_get_surface_element_at(mapX >> 5, mapY >> 5);
-    if (surfaceMapElement == NULL) {
-        return;
-    }
-    if (surfaceMapElement->properties.surface.ownership & 0xF0) {
-        return;
-    }
-
-    mapX = mapX + 16 + (word_981D6C[direction].x * 15);
-    mapY = mapY + 16 + (word_981D6C[direction].y * 15);
```

Comment:
While working on this pull request I began to have some issues with getting map_get_path_element_at to work correctly in the `map_place_peep_spawn` function, and this portion got removed. 

I'm not sure of the original intention of `word_981D6C` or if that portion was even necessary, _and come to think of it I never did test this in various map rotations/directions_, as after the troubles I was just happy with getting map_get_path_element_at to work properly.

In a moment, I'm going to test add
```
sint32 mapXOld = mapX + 16 + (word_981D6C[direction].x * 15);		
sint32 mapYOld = mapY + 16 + (word_981D6C[direction].y * 15);
assert(mapX == mapXOld && mapY == mapYOld);
```
and see if the values are different in various map rotations, directions, and sides of the world. Should there be no assertion errors, should I push the above code in with a comment?